evap/evaluation/models.py

```diff
@@ -759,15 +771,20 @@ def can_publish_rating_results(self):
     def ready_for_editors(self):
         pass
 
-    @transition(field=state, source=State.PREPARED, target=State.EDITOR_APPROVED)
+    @transition(
+        field=state,
+        source=State.PREPARED,
+        target=State.EDITOR_APPROVED,
+        conditions=[lambda self: self.has_valid_main_language],
```

Comment:
Should we enforce this with a constraint? I think it would be a sensible thing to do, but it would also mean changing a lot of tests

Reply:
Constraint would be good, but let's pull this out from this PR, it's fine to merge without the constraint I think.

I think we have some other fields as well that need to be set up in a specific way in order to get "sensible" instances. In the long run, it would be nice to have those filled automatically to simplify tests. I could see us switching to [recipes](https://model-bakery.readthedocs.io/en/latest/recipes.html) or maybe adding some custom `model_bakery_default` members to the fields (but we would need to [patch](https://model-bakery.readthedocs.io/en/latest/how_bakery_behaves.html#customizing-baker) model bakery to respect them). Not a trivial thing, I don't want to mix it up in this PR.